backend/test/modules/doubles-tournament/core/application/BookTableAndPublishEventAfterCallingMatch.spec.ts

```diff
@@ -0,0 +1,187 @@
+import { FromListIdGeneratorStub } from '../../../../test-support/shared/core/FromListIdGeneratorStub';
+import { testDoublesTournamentsModule } from './TestDoublesTournamentsModule';
+import { CallMatch } from '../../../../../src/modules/doubles-tournament/core/application/command/CallMatch';
+import { MatchWasCalled } from '../../../../../src/modules/doubles-tournament/core/domain/event/MatchWasCalled';
+import { BookTournamentTable } from '../../../../../src/modules/tournament-tables/core/application/command/BookTournamentTable';
+import { InMemoryCommandBus } from '../../../../../src/shared/infrastructure/core/application/command/InMemoryCommandBus';
+import { CommandHandler } from '../../../../../src/shared/core/application/command/CommandHandler';
+import { CommandResult } from '../../../../../src/shared/core/application/command/CommandResult';
+import Failure = CommandResult.Failure;
+
+describe('Execute command CallMatch', () => {
+  const team1Id = 'Team1Id';
+  const team2Id = 'Team2Id';
+  const tournamentId = 'TournamentId';
+  const matchNumber = 1;
+  const tableNumber = 1;
+  const currentTime = new Date();
+  const entityIdGen = FromListIdGeneratorStub([team1Id, team2Id]);
+
+  it('Then publish event and execute BookTournamentTable command from different module | everything success', async () => {
```

Comment:
Jesli już opisujesz `everythin success` to fajne są na to nazwy jak : `success scenario` albo `happy path`.